src/components/Stepper.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import React from "react";
+import { ArrowRightIcon } from "@heroicons/react/24/solid";
+import Button from "@/components/Button";
+import { cn } from "@/lib/utils";
+// TODO FOR LATER REFACTOR:
+//  - setting the width for the stepper: do we want to specify the width or base it off parent's width
+//  - variant colors for stepper button
+
+// chips or blocks
+type StepperStyle = "chips" | "icons";
+type StepperStatus = "completed" | "current" | "remaining";
+
+export interface SteppersItem {
+  icon?: string;
+  status: StepperStatus;
+  onClickEvent?: () => void;
+  name: string;
+}
+
+export interface StepperProps {
+  styleType: StepperStyle;
+  stepperWidth: string;
+  steppers: SteppersItem[];
+}
+
+export default function Stepper({
+  styleType,
+  stepperWidth,
+  steppers,
+}: StepperProps) {
+  return (
+    <div className={`${stepperWidth}`}>
+      <div className="flex">
+        {steppers.map((step) => {
+          const { icon, status, onClickEvent, name } = step;
+
+          let chipBgColor: string = "bg-neutral";
+          if (status === "current") chipBgColor = "bg-primary-content";
+          if (status === "completed") chipBgColor = "bg-success-content";
+
+          return (
+            <div
+              key={name}
+              className={cn(
+                "flex flex-1 last:flex-none relative last:after:contents-none after:absolute after:content-[''] after:h-[7px] after:w-full after:top-1/2 after:-translate-y-1/2 after:-z-10",
+                "after:bg-base-100"
+              )}
+            >
+              {styleType === "icons" ? (
+                <Button
+                  onClick={onClickEvent}
+                  className={cn("text-center", styleType === "icons" && "flex")}
+                  variant={status === "completed" ? "secondary" : "neutral"}
```

Comment:
Same thing here, I would prefer being able to set the colors from outside the component as props

Reply:
Actually, let me get clarification on what is wanted here. I'll let you know tomorrow

Reply:
colors will be hardcoded

neutral variant colors. You may need to override the styles and not use the variant, not sure

Reply:
i'm going to omit the variant and add the colors in as classes instead